package.json

```diff
@@ -50,11 +50,12 @@
     "jwt-decode": "^2.2.0",
     "nodemailer": "^6.4.16",
     "oauth": "^0.9.15",
+    "pkce-challenge": "^2.1.0",
     "preact": "^10.4.1",
     "preact-render-to-string": "^5.1.7",
     "querystring": "^0.2.0",
     "require_optional": "^1.0.1",
-    "typeorm": "^0.2.24"
+    "typeorm": "^0.2.29"
```

Comment:
Fixes an issue with one of its dependencies, Highlight.js version 9 (which is not supported anymore). Constantly got an annoying warning in the terminal.